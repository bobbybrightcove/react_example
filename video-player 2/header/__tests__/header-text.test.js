import React from 'react';
import renderer from 'react-test-renderer';
import HeaderText from '../header-text';


describe('HeaderText', () => {
  it('renders according to valid snapshot', () => {
    const component = renderer.create(
      <HeaderText title="John Carter" />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
