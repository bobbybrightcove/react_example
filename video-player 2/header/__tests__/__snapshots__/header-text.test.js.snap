// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderText renders according to valid snapshot 1`] = `
<div
  className="headerText"
>
  <p>
    John Carter
  </p>
</div>
`;
